shell/platform/darwin/ios/platform_view_ios.mm

```diff
@@ -188,14 +190,6 @@ new AccessibilityBridge(static_cast<FlutterView*>(owner_controller_.get().view),
   [owner_controller_.get() platformViewsController]->Reset();
 }
 
-fml::scoped_nsprotocol<FlutterTextInputPlugin*> PlatformViewIOS::GetTextInputPlugin() const {
```

Comment:
We might want to clean this a bit more, but it was strange having this in separately maintained lifecycles in various owners